.changeset/short-mice-itch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@firebase/firestore': minor
+---
+
+Expanded `Firestore.WithFieldValue<T>` to include `T`. This allows developers to delegate `WithFieldValue<T>` inside a wrappers of type `T` to avoid exposing Firebase types beyond Firebase-specific logic.
```

Comment:
I think "a wrappers" should just be "wrappers"

Reply:
done. thanks for catching